src/components/EnvelopeList.vue

```diff
@@ -124,6 +125,7 @@ import MoveModal from './MoveModal'
 import { matchError } from '../errors/match'
 import NoTrashMailboxConfiguredError
 	from '../errors/NoTrashMailboxConfiguredError'
+import DragEventBus from '../directives/drag-and-drop/util/DragEventBus'
```

Comment:
```suggestion
import dragEventBus from '../directives/drag-and-drop/util/DragEventBus'
```

Because it's an instance/object, not a type